lib/viz_import/volume_chapter_importer.rb

```diff
@@ -0,0 +1,78 @@
+class VolumeChapterImporter
+  class ImportFiles
+    attr_reader :data
+
+    def initialize(filename)
```

Comment:
This logic might be a lot cleaner if you divvy it up into smaller `VizVolume`, `VizDump`, and `VizChapter` classes

As it is, I'm having a lot of trouble following the flow